Include/Acidanthera/Library/OcBootManagementLib.h

```diff
@@ -785,9 +790,9 @@ struct OC_PICKER_CONTEXT_ {
   //
   OC_CUSTOM_DESCRIBE         CustomDescribe;
   //
-  // Context to pass to CustomRead and CustomDescribe, optional.
+  // Storage context.
   //
-  VOID                       *CustomEntryContext;
+  OC_STORAGE_CONTEXT         *StorageContext;
```

Comment:
I already abused the fact that CustomEntryContext contained OC_STORAGE_CONTEXT in the flavour code, I thought it was better to make it explicit.

Reply:
Fine to me.